imagehash.go

```diff
@@ -54,6 +54,9 @@ func (h *ImageHash) Bits() int {
 
 // Distance method returns a distance between two hashes.
 func (h *ImageHash) Distance(other *ImageHash) (int, error) {
+	if other == nil {
+		return -1, errors.New("other should not be nil")
+	}
```

Comment:
```suggestion
var errNoOther = errors.New("other should not be nil")

// Distance method returns a distance between two hashes.
func (h *ImageHash) Distance(other *ImageHash) (int, error) {
	if other == nil {
		return -1, errNoOther
	}
```

This is example code, and it looks like the error global variable should be declared at the top if possible.